code/game/turfs/simulated/wall_attacks.dm

```diff
@@ -26,13 +26,13 @@
 
 /turf/simulated/wall/proc/fail_smash(var/mob/user, var/multiplier = 1)
 	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN*2.5)
-	to_chat(user, SPAN_DANGER("You smash against the wall!"))
+	to_chat(user, "<span class='danger'>You smash against the wall!</span>")
```

Comment:
Why go from using SPAN_DANGER to not? I assume it's due to a merge while resolving conflicts, I'd double check the previously conflicted files.